infra/cmake/packages/FarmhashSourceConfig.cmake

```diff
@@ -7,10 +7,12 @@ function(_FarmhashSource_import)
   nnas_include(ExternalSourceTools)
   nnas_include(OptionTools)
 
-  # NOTE TensorFlow 1.7 downloads farmhash from the following URL
-  envoption(FARMHASH_URL https://github.com/google/farmhash/archive/816a4ae622e964763ca0862d9dbd19324a1eaf45.tar.gz)
+  # NOTE TensorFlow 1.12 downloads farmhash from the following URL
+  #      TensorFlow 1.13.1 downloads farmhash from the following URL
+  #      TensorFlow 2.2 downloads farmhash from the following URL
```

Comment:
(Q) As I know, we use 2.3.0-rc0 as our experiment with TF2.x. Do we use 2.2 anywhere?